src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable,AreaCalculator {
+    private String color;
```

Comment:
```suggestion
    private Color color;
```
let's use enum instead of string